test/blocks/section-metadata/section-meta.test.js

```diff
@@ -83,6 +83,9 @@ describe('Section Metdata', () => {
     const sm = sec.querySelector('.section-metadata');
     await init(sm);
     expect(main.lastElementChild.classList.contains('hide-sticky-section')).to.be.true;
+    window.scrollTo(0, document.body.scrollHeight);
+    await delay(500);
```

Comment:
Could we use `clock.runAllAsync();` or something similar to avoid the wait time?